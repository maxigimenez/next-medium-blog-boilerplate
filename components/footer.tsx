import React from 'react';
import config from '@/config';

export const Footer = () => (
  <div className="container mt-5">
    <footer className="bg-white border-top p-3 text-muted small">
      <div className="row align-items-center justify-content-between">
        <div>
          <span className="navbar-brand mr-2">
            <strong>{config.name}</strong>
          </span>
          Copyright &copy; 2020 . All rights reserved.
        </div>
        <div>
          Made with
          <a
            target="_blank"
            className="font-weight-bold"
            href="https://github.com/maxigimenez/next-medium-blog-boilerplate"
            rel="noreferrer"
          >
            Next.js Blog Boilerplate
          </a>
          by
          <a href="https://twitter.com/@gmaxi_" className="font-weight-bold" target="_blank" rel="noreferrer">
            @gmaxi_
          </a>
        </div>
      </div>
    </footer>
  </div>
);
