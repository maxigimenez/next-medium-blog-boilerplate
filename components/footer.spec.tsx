import React from 'react';
import { shallow } from 'enzyme';
import { Footer } from './footer';

describe('footer', () => {
  describe('snapshots', () => {
    test('render', () => {
      const component = shallow(<Footer />);
      expect(component).toMatchSnapshot();
    });
  });
});
