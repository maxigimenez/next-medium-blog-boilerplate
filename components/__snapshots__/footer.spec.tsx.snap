// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer snapshots render 1`] = `
<div
  className="container mt-5"
>
  <footer
    className="bg-white border-top p-3 text-muted small"
  >
    <div
      className="row align-items-center justify-content-between"
    >
      <div>
        <span
          className="navbar-brand mr-2"
        >
          <strong>
            Demo Blog
          </strong>
        </span>
        Copyright Â© 2020 . All rights reserved.
      </div>
      <div>
        Made with
        <a
          className="font-weight-bold"
          href="https://github.com/maxigimenez/next-medium-blog-boilerplate"
          rel="noreferrer"
          target="_blank"
        >
          Next.js Blog Boilerplate
        </a>
        by
        <a
          className="font-weight-bold"
          href="https://twitter.com/@gmaxi_"
          rel="noreferrer"
          target="_blank"
        >
          @gmaxi_
        </a>
      </div>
    </div>
  </footer>
</div>
`;
